toolkit/tools/pkg/imagecustomizerlib/shrinkfilesystems.go

```diff
@@ -93,16 +102,23 @@ func getStartSectors(imageLoopDevice string, partitionCount int) (matchStarts []
 }
 
 // Get the filesystem size in sectors.
-func getFilesystemSizeInSectors(resize2fsOutput string, imageLoopDevice string) (filesystemSizeInSectors int, err error) {
+func getFilesystemSizeInSectors(resize2fsStdout string, resize2fsStderr string, imageLoopDevice string,
+) (filesystemSizeInSectors int, err error) {
+	if strings.Contains(resize2fsStderr, "Nothing to do!") {
+		return -1, err
```

Comment:
I find the -1 a bit weird. Since you already return an error then maybe return 0 here?

Reply:
This is supposed to be `return -1, nil`. Fixed.